.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Report a bug and help us improve
+title: "[BUG]"
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1.
+2.
+3.
+
+**Environment**:
+- multus-dynamic-networks-controller version: N/A
+- Kubernetes version (use `kubectl version`): N/A
+- Network-attachment-definition: N/A
+- OS (e.g. from /etc/os-release): N/A
+- Which runtime is being used: N/A
```

Comment:
I have no idea how to find out. Is there any go-to command to find that information?

Reply:
Hm, I'll request for the controller configuration instead - you have to put the CRI socket there. 

I think the runtime must be inferred from there.

Reply:
@phoracek ^